masterlist.yaml

```diff
@@ -34013,3 +34020,21 @@ plugins:
         util: '[TES4Edit v4.0.4](https://www.nexusmods.com/oblivion/mods/11536)'
         itm: 4
         udr: 40
+
+  - name: 'DLCFrostcragCoblAlchemyTable.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/52608/'
+        name: 'Frostcrag Spire Cobl Alchemy Station'
+    req: [ *Cobl ]
+    after: [ 'DLCFrostcrag - Unofficial Patch.esp' ]
+    tag: [ Scripts ]
+    dirty:
+      # Version: 0.9
+      - <<: *quickClean
+        crc: 0x802681D8
+        util: '[TES4Edit v4.0.4](https://www.nexusmods.com/oblivion/mods/11536)'
+        udr: 4
```

Comment:
```suggestion
```

The plugin already has Cobl as a master, so the requirement is redundant. `DLCFrostcrag - Unofficial Patch.esp` is in an earlier-loading group, so the after is also redundant. V1.0 of the plugin has the tag in its description, so that's also redundant. We also don't keep cleaning data for older versions since that'd bloat the masterlists.